iotdb-core/node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -649,6 +649,16 @@ data_replication_factor=1
 # Datatype: int
 # compaction_write_throughput_mb_per_sec=16
 
+# The limit of read throughput merge can reach per second
+# values less than or equal to 0 means no limit
+# Datatype: int
+# compaction_read_throughput_mb_per_sec=40
```

Comment:
Also, notice the difference between Mb and MB here.

Reply:
The parameter `compaction_write_throughtput_mb_per_sec` has existed in past version and been used extensively. To ensure consistent name style,  the parameter is `compaction_read_throughput_mb_per_sec`. To distinguish the unit, I think I can add the unit in comment.